core/controllers/admin.py

```diff
@@ -273,7 +273,7 @@ def post(self):
                 result = {
                     'opportunities_count': opportunities_count
                 }
-            elif action == 'update_feature_flag_rules':
+            elif self.payload.get('action') == 'update_feature_flag_rules':
```

Comment:
Why is this changed and the others not?

Reply:
Fixed, probably got there when doing some merge.